docs/install-bitbucket-server.md

```diff
@@ -57,7 +64,7 @@
               JF_GIT_USERNAME: ""
   
               # [Mandatory]
-              # Bitbucket accesses token with the following permissions 
+              # Bitbucket accesses token with write repository permissions
```

Comment:
accesses --> access

Search for "accesses" in the emtire code. We should fix it in other places as well.

Reply:
done